cocos/2d/components/graphics.ts

```diff
@@ -262,12 +262,13 @@ export class Graphics extends UIRenderer {
 
     public onLoad () {
         super.onLoad();
-        this.model = director.root!.createModel(scene.Model);
```

Comment:
`this.model` is a public property, will it be accessed in other modules?
If it does, it will be `undefined` on JSB which will break the compatibility.